lib/views/variant-view.dart

```diff
@@ -76,84 +76,95 @@ class SortablePage extends StatefulWidget {
 }
 
 class _SortablePageState extends State<SortablePage> {
-  List<String> headerLabel = ['accession', 'geographical location', 'date collected', 'generated', 'pinned'];
+  List<String> headerLabel = ['accession', 'geographical location', 'date collected', 'generated', 'pinned', 'selected'];
```

Comment:
For now this is fine, but need to hammer out the difference between pinned and selected later on - probably don't want to have both - thinking maybe a button that people can click to save an item rather than one that people check - i.e. user clicks the Save button (maybe some icon) and a pop-up says "Variant added to Saved list" or something - this isn't in scope of this but good thing to think about

Reply:
Yeah, something for later consideration

Reply:
Looks good to me! The difference between pin and selection does need to be ironed out, but I think this change helps a lot moving forward